packages/EasyErrorHandler/src/Bridge/Symfony/Command/AnalyzeErrorCodesCommand.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Command;
```

Comment:
Pls change namespace to `Commands` to be consistent.

Reply:
Done.

Reply:
Done.